app/views/layouts/application.html.erb

```diff
@@ -135,5 +137,8 @@
     <%= link_to 'GitHub', "https://github.com/aqora-io/quantumnews" %>
   </footer>
   <span id="iab-pcm-sdk"></span><span id="iab-autofill-sdk"></span>
+  <% if Rails.application.config.cookie_consent_config_js.present? %>
```

Comment:
I don't know if this script is also blocking google analytics automatically, but in the case that it is, it probably needs to be loaded before the gtag.js?

Reply:
@jpopesculian I can confirm it works this way for the main site. Adding `type="text/plain" data-category="marketing"` will put them in some hook to be loaded once confirmed

Reply:
i mean, ideally these kind of scripts should be deferrable though. or at least loaded at the very end of the HTML document

Reply:
You mean all of it should be put at the end of the `body`? doing it according to the doc [here](https://cookieconsent.orestbida.com/essential/getting-started.html#html)

Reply:
ah, you mean putting the gtag thingy behind it?

Reply:
moved it @jpopesculian